biojava-core/src/main/java/org/biojava/nbio/core/util/FileOperationUtils.java

```diff
@@ -0,0 +1,151 @@
+package org.biojava.nbio.core.util;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.channels.FileChannel;
+import java.nio.file.*;
+import java.nio.file.attribute.BasicFileAttributes;
+
+/**
+ * @author Sreejith Nair
+ */
+public class FileOperationUtils {
+    /**
+     * @param src
+     * @param dst
+     * @throws IOException
+     */
+    @SuppressWarnings("resource")
+    public static void copy(File src, File dst) throws IOException {
```

Comment:
Since this PR is all about cleaning up, how about taking this opportunity to remove this method and replace it by the one provided in the java library (see TODO comment in original javadocs above)